chore!: Change graceful shutdown username to graceful-shutdown-user (#573)
stackabletech/trino-operator: CHANGELOG.md

- Support row filters and column masks in Rego rules ([#559]).

### Changed

> - BREAKING: Change the username that triggers graceful shutdown from `admin` to `graceful-shutdown-user` for more expressiveness (e.g. in the Trino policies).

Reviewer: suggestion: Two small suggestions regarding the changelog text.

```suggestion
- BREAKING: Change the username which triggers graceful shutdown from `admin` to `graceful-shutdown-user` for more expressiveness (e.g. in the Trino policies).
```

>   This is breaking as users needs to make sure the user `graceful-shutdown-user` has the needed permissions to initiate a graceful shutdown ([#573]).

Reviewer: ```suggestion
  This is a breaking change because users need to ensure that the user `graceful-shutdown-user` has the required permissions to initiate a graceful shutdown ([#573]).
```
